curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md

```diff
@@ -55,7 +55,7 @@ tests:
 <div id='js-seed'>
 
 ```js
-function solve24 (numStr) {
+function solve24 (numStr){
```

Comment:
This is more of a stylistic thing, but could you change this and the solution below to the following?:

`function solve24(numStr) {`

A lot of these challenges have a pesky space between the function name and the open parenthesis (`function solve24 (numStr) {`) for some reason. Might have been due to the scraper when these were originally ported to the fCC Arcade Mode.

Reply:
Ahh, I see, making changes